source/common/network/address_impl.h

```diff
@@ -312,6 +330,13 @@ class PipeInstance : public InstanceBase {
   }
   absl::string_view addressType() const override { return "default"; }
 
+  void abslHashValue(absl::HashState state) const override {
+    absl::HashState::combine(absl::HashState::combine_contiguous(std::move(state),
+                                                                 pipe_.address_.sun_path,
+                                                                 sizeof(pipe_.address_.sun_path)),
```

Comment:
I think this should be `address_length_`?

Reply:
address_length_ is only set for abstract namespace. For non abstract namespace, it doesn't set 
https://github.com/envoyproxy/envoy/blob/357bf0016ffa0102ed1fc3c51687eb23b9d2305d/source/common/network/address_impl.cc#L381-L389

if use the `sizeof(pipe_.address_.sun_path)`, it will be safe if the `pipe_.address_.sun_path` set to zero by `memset`. But yea, there are PipeInstance's constructs method accept `sockaddr_un* address` as the parameter. It can guarantee the memory is set as zero.

Maybe I can use address_length_ for abstract namespace. The use `strlen` for non abstract namespace.

Reply:
Yeah, I was looking at equality or addressAsString I think, and it's possible that usage is wrong there.